include/trajopt/trajectory/HolonomicTrajectorySample.h

```diff
@@ -32,6 +34,12 @@ class TRAJOPT_DLLEXPORT HolonomicTrajectorySample {
   /// The angular velocity.
   double angularVelocity = 0.0;
 
+  /// The force on each module in the X direction.
+  std::vector<double> moduleForcesX{};
+
+  /// The force on each module in the Y direction.
+  std::vector<double> moduleForcesY{};
```

Comment:
```suggestion
  std::vector<double> moduleForcesX;

  /// The force on each module in the Y direction.
  std::vector<double> moduleForcesY;
```

Reply:
This change will prevent the default constructor from being `constexpr` it seems.

Reply:
![image](https://github.com/SleipnirGroup/TrajoptLib/assets/25559812/b78c0467-f4e3-4727-bcef-42a8df5bb815)